phpunit/class-wp-theme-json-test.php

```diff
@@ -4916,6 +5020,60 @@ public function test_get_block_background_image_styles() {
 		$this->assertSameCSS( $verse_styles, $theme_json->get_styles_for_block( $verse_node ), 'Styles returned from "::get_styles_for_block()" with default core/verse background styles as string type do not match expectations' );
 	}
 
+	/**
+	 * Testing background dynamic properties in theme.json.
+	 */
+	public function test_get_resolved_background_image_styles() {
```

Comment:
Good question. 

I was wondering whether `get_property_value` deserves testing on its own - I don't think it would hurt, since it fits the definition of a "unit" in testing. 

To answer your question, yeah, I wanted to also cover the other properties since none of them are resolved in trunk at the moment.